frappe/public/js/frappe/form/controls/geolocation.js

```diff
@@ -0,0 +1,183 @@
+frappe.ui.form.ControlGeolocation = frappe.ui.form.ControlData.extend({
+	make_wrapper() {
+		// Create the elements for map area
+		this._super();
+
+		let $input_wrapper = this.$wrapper.find('.control-input-wrapper');
+		this.map_area = $(
+			`<div class="map-wrapper border">
+				<div id="` + this.df.fieldname + `" style="min-height: 400px; z-index: 1; max-width:100%"></div>
```

Comment:
avoid id, put styles in form.less

Reply:
id is required for Leaflet Map
http://leafletjs.com/reference-1.2.0.html#map
```
this.map = L.map(this.df.fieldname).setView([19.0800, 72.8961], 13);
```